language-service/language_service/service/vocab/cedict/__init__.py

```diff
@@ -0,0 +1,14 @@
+import json
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in public package](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)